### DISCUSSION TITLE
open banner overlay when the page is loaded with fancybox 4

### DISCUSSION THREAD

**Discussion by Bellia86** [fancyapps/ui#126] [Q&A]
I would like to update to the latest version of fancybox 4 the javascript code of my banner overlay that opens when the page is loaded. I don't understand where I'm wrong.

```
<script type="text/javascript">
(function ($) {
  if(!Cookies.get('overlay2533'))
  {
    $(document).ready(function () {
      Fancybox.show([
        {
          src : "#fancybox-overlay",
          type : "image"
        }
      ]);
    });
    var date = new Date();
    date.setHours(24,0,0,0);

    Cookies.set("overlay2533", "1", { expires: new Date( date ) });
  }
})(jQuery);
</script>

<div id="fancybox-overlay" style="display:none;">
  <href="htts://....." target="_blank"><img src="https://....."  alt="" /></a>
</div>
```

**Comment by fancyapps** [comment]
Hi,

Maybe remove `$(document).ready(function () {})` part, but it is just a guess, a live demo would help to give a proper answer.
Also, do you really have `type : "image"`? That looks like inline content.

**Comment by Bellia86** [comment]
this is the page where I am doing the tests:
https://jsfiddle.net/t2xwsfrd/1/
Thanks

**Comment by Bellia86** [comment]
I solved it by simply editing
`type: "inline"`
